### PR TITLE
modalOfsignUp

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by MaryPG21 — APPROVED** [review]
That's OK.

**Review by samikia — APPROVED** [review]
great

File: frontend/src/components/DropDown.jsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import React from "react";
 import { useState } from "react";
+import Modal, { handleClick } from "./modalSignup";
```
Comment (mhmdrezanaderi) [inline frontend/src/components/DropDown.jsx:3]: I think we can remove this line

**Review by mhmdrezanaderi — APPROVED** [review]
Thanks a lot

**Pull request merged**